interfaces/builtin/network_test.go

```diff
@@ -24,12 +24,14 @@ import (
 
 	"github.com/ubuntu-core/snappy/interfaces"
 	"github.com/ubuntu-core/snappy/interfaces/builtin"
+	"github.com/ubuntu-core/snappy/snap"
 )
 
 type NetworkInterfaceSuite struct {
-	iface interfaces.Interface
-	slot  *interfaces.Slot
-	plug  *interfaces.Plug
+	iface         interfaces.Interface
+	slot          *interfaces.Slot
+	otherSnapSlot *interfaces.Slot
```

Comment:
This file is very similar to the network binding one, so all comments above apply here too.